packages/react-vapor/src/components/input/Input.tsx

```diff
@@ -243,3 +248,24 @@ export class Input extends React.Component<IInputProps, IInputComponentState> {
         );
     }
 }
+
+const mapStateToProps = (state: IReactVaporState, ownProps: IInputProps): IInputStateProps => {
+    const input = InputSelectors.getInput(state, {id: ownProps.id});
+    return {
+        valid: input && input.valid,
+        value: input && input.value,
+        disabled: input && input.disabled,
+    };
+};
+
+const mapDispatchToProps = (dispatch: IDispatch, ownProps: IInputProps): IInputDispatchProps => ({
+    onRender: (value: string = '', valid = true, disabled = false) =>
+        dispatch(addInput(ownProps.id, value, valid, disabled)),
+    onDestroy: () => dispatch(removeInput(ownProps.id)),
+    onChange: (value: string, valid = true) => {
+        dispatch(changeInputValue(ownProps.id, value, valid));
+        ownProps.changeDirtyState?.(value);
+    },
+});
+
+export const InputConnected = connect(mapStateToProps, mapDispatchToProps, ReduxUtils.mergeProps)(Input);
```

Comment:
Is it out of scope to refactor to redux hooks?

Reply:
Yes, that input is going into the garbage can anyway 😅